### PR TITLE
Insert admin user via web ui

### PR DESCRIPTION
**Pull request by binary1230**
Does anyone hate this idea? I feel like I should hate myself for writing this.  Or it's brilliant.  I'm not sure.

I made it so if you visit /accounts/insert_test_admin, it will do the same thing as 'sep insert_admin'

It will ONLY let you do this if there are zero admin users on the install.  Otherwise it will deny it.  So if you had an existing install and someone tried to visit this URL, it will deny them from trying to create this.

Eventually this could be re-worked into some kind of 'first time setup' UI or something like that.  It's a bit beyond my scope here and probably needs a lot of thinking about.

dom-unstable in dockerland is pointing at this if anyone wants to check it out for themselves.  I poked it with a stick a few times it appears to behave.

There's an actual reason I wrote this too.  'docker exec sep insert_admin' doesn't have access to the DB env vars that sideboard was started with, so it can't work.  There's an easy fix there which is simply to have it connect to a host named 'db' on the psql standard port and assume a certain DB username/pass.  Separate thing.

### CODE REVIEW

**Comment by thaeli** [comment]
Love the concept, want to look closer at the implementation just to be sure
we're not opening security holes.

File: uber/sep_commands.py [hunk 1]
```diff
@@ -14,19 +14,10 @@ def print_config():
 @entry_point
 def insert_admin():
     with Session() as session:
```
Comment (EliAndrewC) [inline uber/sep_commands.py]: Since you're no longer doing database stuff in this method, you no longer need the ``with Session() as session`` block.  Doesn't Doesn't hurt to leave it in, it's just unnecessary.

Comment (EliAndrewC) [inline uber/sep_commands.py]: Oh wait, I just realized you're passing in the session to the function.

In general, whenever we have a function which takes a session object as its first argument, we typically put that in the ``SessionMixin`` class in ``models.py`` so that we can call it as a session method.

Alternatively, you could just move the session instantiation into the utility function itself.

Not a huge deal, since this will work as-is, it'sj ust not how we're doing things elsewhere in the codebase.

**Comment by EliAndrewC** [comment]
> There's an actual reason I wrote this too. 'docker exec sep insert_admin' doesn't have access to the DB env vars that sideboard was started with, so it can't work.

I haven't yet had time to even spin up a Docker container, so pardon my ignorance: does this mean that we can basically no longer run ``sep`` commands which affect the database period?

**Comment by binary1230** [comment]
at the moment, yea, the sep stuff (when run with 'docker exec') is borked because it isn't currently passed the same env vars that docker-compose is using like the DB hostname.

there's a workaround which would look something like:
docker exec -i CONTAINER_ID /bin/bash -c "export DB_HOST=1.2.3.4 your_cmd"

Though, also, there's an alternative solution which docker-compose does which we can take advantage of.  When you run a docker container with docker-compose, it will modify the /etc/hosts file on the fly and add an entry like:

```bash
docker exec -ti docker_web_1 /bin/bash  # run a bash shell in our existing container
root@48b212804cf2:/mnt/app/uber/sideboard# cat /etc/hosts
...
172.17.0.33     db 269cd5427866 docker_db_1
```

Thus, in our current setup, 'db' always resolves to the postgres DB.  If we just use that and don't bother to use the ENV vars in c.SQL_ALCHEMY_URL, then the sep commands will work once again.  This is, I think, a recommended approach when using docker-compose.

TLDR: 
===
* sep commands still work but you may need to pass them ENV vars right now, slight annoyance
* I had wanted for a while the ability to get a fully setup container without having to login to the host and run sep commands, so this is I think the final step needed there

**Comment by EliAndrewC** [comment]
Gotcha.  It sounds like there's a path to be able to use sep commands in the future, which was my main concern, so I'm +1 on this change.

File: uber/sep_commands.py (hunk 1 above)
Comment (binary1230) [inline uber/sep_commands.py]: ok so you're saying make the method be SessionMixin::insert_test_admin() instead of a standalone function, and make the sep command look like:

```python
@entry_point
def insert_admin():
    with Session() as session:
        session.insert_test_admin()
```

and the web call look like

```
session.insert_test_admin()
```

**Comment by binary1230** [comment]
actually hey i might have been lying about the env vars, they are definitely present in 'docker exec', I'm curious why that wasn't working.....  I still want to do this change but let me check that out...

File: uber/sep_commands.py (hunk 1 above)
Comment (EliAndrewC) [inline uber/sep_commands.py]: Right, that's how we're doing things elsewhere.

**Comment by binary1230** [comment]
so actually (this is getting way off-topic from this PR now) the env vars are definitely getting passed to sep.  It's choking on on unboundcolumnerror when it tries to access "c.ATTENDEE_BADGE". let me check if the new recent DB changes we did affect when /how it tries to connect to the DB and if that's somehow not working for sep insert_admin just by initializing Session. 

DEBUGGER AWAY

**Comment by binary1230** [comment]
AH, yea that's it.  we're not calling Session.initialize_db() in sep commands anymore because we only call it when the sideboard server starts (which doesn't happen in sep commands I guess) so it never connects.

**Comment by binary1230** [comment]
booya. that fixed it, so disregard all that env var stuff, it all now works again.  

Also, did the CR suggestions and added stuff to the SessionMixin class.

\m/

**Comment by binary1230** [comment]
to be clear, sep commands should work fine from the command line of the docker host like this:
```bash
docker exec -ti docker_web_1 /uber/env/bin/sep insert_admin
```

**Comment by EliAndrewC** [comment]
Sweet, glad to hear it all around!

**Comment by thaeli** [comment]
Actually, you can use Docker's -e switch to just pass env vars in directly. No need to go through a shell.

**Comment by binary1230** [comment]
@thaeli Yea, I thought that would have been the reasonable case too but it appears that 'docker exec' doesn't directly support passing in env vars via -e like 'docker run' does.  I think I saw some PR's on the docker repo where they discussed implementing support for doing that, 'docker exec' is a relatively new command

**Comment by binary1230** [comment]
closing as this functionality was rebased in on https://github.com/magfest/ubersystem/pull/1177

there is some stuff in this branch we might need.  it all got tangled up, untangling now.
